### ISSUE TITLE
Create a events page

### ISSUE THREAD

**Issue by abhijithvijayan** [ceadoor/cea.ac.in#114]
create a page called 
```events.html``` under ```src/news-and-events/```
with headers and footers from 
```src/contact.html```

and include that in ```webpack.js``` accordingly.

Make sure you change the ```title``` in html page to ```Events | CEA```

**Comment by Fedelaus** [comment]
I notice you are including all HTML files currently manually adding them to the plugins list in your `webpack.config.js`. Could I suggest globbing all files and adding them that way, as I cannot see a reason why you may have a html file in your `src/` directory without wanting it in your final build directory. It would remove the requirement to artificially inflate the size of your webpack config file.

If this is an attractive solution to you, I'm happy to implement this in your project? It would be easiest done with `node-glob` (dev dependency) but could be done with flat `fs` too.

**Comment by abhijithvijayan** [comment]
@Fedelaus You saying to group the imports of html files and include in ```webpack.config.js``` ?
if so we'll get to that in a bit. Thanks.

**Comment by ashikmeerankutty** [comment]
@Fedelaus You are right. We never thought that there will be so much html files so we went with this approach and now see that it makes our `webpack.config.js` big. We are using `HtmlWebpackPlugin` to inject the final `css` and `js` files and also change the assets path. We think your approach is the better way so you can work on that. You can go with `node-glob`.
